android-activity/Cargo.toml

```
@@ -9,6 +9,7 @@ repository = "https://github.com/rust-mobile/android-activity"
 documentation = "https://docs.rs/android-activity"
 description = "Glue for building Rust applications on Android with NativeActivity or GameActivity"
 license = "MIT OR Apache-2.0"
+rust_version = "0.64"
```

That's old, did a `0.64` release even exist?

```suggestion
rust_version = "1.64"
```